src/status_im/transport/message/v1/protocol.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im.transport.message.v1.protocol
+  (:require [status-im.utils.config :as config]
+            [status-im.constants :as constants]
+            [status-im.chat.core :as chat]
+            [status-im.transport.message-cache :as message-cache]
+            [status-im.transport.db :as transport.db]
+            [status-im.transport.core :as transport]
+            [status-im.transport.message.core :as message]
+            [status-im.transport.utils :as transport.utils]))
+
+(def ^:private whisper-opts
```

Comment:
Why do we need to use `^:private` here? Does it affect anything?

Reply:
Because everything which is not used outside namespace should be private, doesn't matter if it's function or var.
It's easier for the person who reads the code when he/she immediately knows that no search outside the namespace is needed, because it's not used anywhere else.

Reply:
When you are using this ns from outside this symbol is still visible (because cljs doesn't apply this metadata afaiu) and it can be used without any warnings/errors. So 
```
when he/she immediately knows that no search outside the namespace is needed
```
is not really true statement.

Reply:
it is true if the person know the meaning of private. I think it's just meant to be informative here not to enforce anything since you can't really enforce private in clojure anyway

Reply:
yeah, but person should look at definition in order to figure that it is "private". Which is not necessary in repl, for instance. Whatever, imo, it's not very useful and rather may mislead